server/src/database/seeds/AdminClientSeeder.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Gernzy\Server\Database\Seeds;
+
+use Gernzy\Server\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+
+// use Illuminate\Support\Facades\DB;
+
+class AdminClientSeeder extends Seeder
+{
+    protected $foo = [];
+
+    /**
+     * Seed the application's database.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $userName = 'admin';
```

Comment:
@AlexeiaW So the only problem with this is it exposes the default admin details. Many people implementing the software will end up just using this as a default, like how in WordPress "admin" became the default username. It makes it easier to brute force because you only then need to hit that username rather than trying various combinations. 

So let's randomly generate the username as well, and the email can be "${randomString}@gernzy.com"

Reply:
Ok, made them random